web/src/components/jobs/JobDetailPage.tsx

```diff
@@ -98,6 +99,7 @@ const JobDetailPage: FunctionComponent<IProps> = (props) => {
       <Box mb={2} display={'flex'} justifyContent={'space-between'} alignItems={'center'}>
         <Tabs value={tab} onChange={handleChange} textColor='primary' indicatorColor='primary'>
           <Tab label={i18next.t('jobs.latest_tab')} disableRipple={true} />
+          <Tab label={'Inputs/Outputs'} disableRipple={true} />
```

Comment:
_Minor_: Similarly to datasets, we can shorten `INPUTS/OUTPUTS` for jobs to `I/O`.